core/gesture.js

```diff
@@ -651,9 +651,6 @@ Blockly.Gesture.prototype.handleWsStart = function(e, ws) {
   this.setStartWorkspace_(ws);
   this.mostRecentEvent_ = e;
   this.doStart(e);
-  if (!this.startBlock_ && !this.startBubble_) {
-    this.fireWorkspaceClick_(ws);
-  }
```

Comment:
This is on purpose right? in response to comments on #3956?

Reply:
Yup! I left it in so just removing